doc/nrf/ug_multi_image.rst

```diff
@@ -0,0 +1,39 @@
+.. _ug_multi_image:
+
+Multi-image builds
+##################
+
+In many cases, the firmware that is programmed to a device consists of not only one application, but several separate images, where one of the images (the *parent image*) requires one or more other images (the *child images*) to be present.
+The most common use case for builds consisting of multiple images is an application that requires a bootloader to be present.
+
+In the context of |NCS|, multiple images are required in the following scenarios:
+
+nRF9160 SPU configuration
+   The nRF9160 SiP application MCU is divided into a secure and a non-secure domain.
+   The code in the secure domain can configure the System Protection Unit (SPU) to allow non-secure access to the CPU resources that are required by the application, and then jump to the code in the non-secure domain.
+   Therefore, the nRF9160 samples (the parent image) require the :ref:`secure_partition_manager` (the child image) to be programmed in addition to the actual application.
+
+   See :ref:`zephyr:nrf9160_pca10090` and :ref:`ug_nrf9160` for more information.
+
+MCUboot bootloader
+   The MCUboot bootloader establishes a root of trust by verifying the next step in the boot sequence.
+   This first-stage bootloader is immutable, which means it must never be updated or deleted.
+   However, it allows to update the application, and therefore MCUboot and the application must be located in different images.
+   In this scenario, the application is the parent image and MCUboot is the child image.
+
+   See :ref:`about_mcuboot` for more information.
+   The MCUboot bootloader is used in the :ref:`http_application_update_sample` sample.
+
+In such cases where the firmware consists of several images, you can build each of the images separately and program it to the correct place in flash.
+
+Alternatively, you can build the related images as one solution, starting from the parent image.
+This is referred to as *multi-image build*, and it is the default way how the |NCS| samples are set up.
+When building the parent image, you can configure if a child image should automatically be built and included in the parent image (the default), if a prebuilt HEX file should be included instead of the child image, or if the child image should be ignored.
+If you are creating your own application, see *Building and Configuring multiple images* in :ref:`zephyr:application` for more information about how to turn your application into a parent image that requires specific child images.
+
+In a multi-image build, all images are placed in memory so that they do not overlap.
+This works fine for simple use cases like a bootloader that prepares a device, where there are no further requirements on where in memory each image must be placed.
```

Comment:
But where does this hardcoding happen? It is not described in the multi-image build section in Zephyr ...

Reply:
It's implied :/.

Before partition manager we had to set the kconfig `CONFIG_FLASH_LOAD_OFFSET` manually for each image.

Reply:
Like this?

In a multi-image build, all images are placed in memory so that they do not overlap.
The flash start address for each image must be specified by :option:`CONFIG_FLASH_LOAD_OFFSET`.
Hardcoding the image locations like this works fine for simple use cases ...

Reply:
Great! I would change
"are placed" -> "must be placed" and
"specified by" -> "specified by e.g."